first_boot-displaymanager-20160805.json

```diff
@@ -0,0 +1,15 @@
+{
+  "properties": [],
+  "tags": [
+    "displaymanager"

```

Comment:
Indeed the gdm needle does work with my test. I simply can't find any existing needle tagged with "rootpassword*" though, which I guess is the only other one you find suspicious?